config/mongo.go

```diff
@@ -57,3 +63,43 @@ func LoadMongoDBFromEnv() *MongoDB {
 	}
 	return &mongo
 }
+
+func (m *MongoDB) mustMode(s *mgo.Session) {
```

Comment:
Generally the 'Must' prefix is used when the function being called has an opportunity for error and it designates that the function will 'panic' if the error occurs.

Since there are no possible errors in setting the mode and server selections, we can probably just call the funcs `setMode` and `setSelectServers`.